### PR TITLE
Improvements/fixes to readme

### PR DESCRIPTION
**Pull request by Ruttie2006**

### CODE REVIEW

**Comment by Ruttie2006** [comment]
wait is gist a word?

**Comment by Ruttie2006** [comment]
Apparently gist is something from github.
Anyway still better to do list, to avoid confusion?
You guys can choose.

**Comment by Extremelyd1** [comment]
While you are at it, the mods folder is platform dependent, so perhaps include a section that mentions the exact paths for all platforms and then refer to it whenever it is needed. For example, in the section about dependencies you can refer to it to indicate where these DLLs are.

**Comment by Ruttie2006** [comment]
i don't know where the mods folder is for linux, i'm still waiting for a response in the hk discord.
will also add the paths to the dependencies

File: README.md [hunk 1]
```diff
@@ -93,9 +102,20 @@ Moreover, the latest announcements will be posted there.
 HKMP can also be built from scratch.
 This requires a few dependencies from the Hollow Knight game and the modding API.
 Namely, the following dependencies should be added as referenced assemblies from **the modding API**:
+```
 - Assembly-CSharp.dll (modified by the modding API)
-
+```
 And the following assemblies should be added as references from **the Hollow Knight game/Unity**:
+
+   i. All the files here should be in the directory:
```
Comment (Extremelyd1) [inline README.md:110]: Why does this sentence start with "i.", there is no enumeration here right?

File: README.md [hunk 2]
```diff
@@ -8,7 +8,15 @@ However, as it seemed to be discontinued, I decided to rework it entirely and ad
 
 ## Install
 The mod works through the [Hollow Knight Modding API](https://github.com/seresharp/HollowKnight.Modding) (a getting started guide can be found [here](https://radiance.host/apidocs/Getting-Started.html)).
-After installing the API, this mod can be installed by dropping the compiled DLL into your mods folder, which can be found in your Steam installation: `<steam>/Hollow Knight/hollow_knight_Data/Managed/Mods/`.
+After installing the API, this mod can be installed by dropping the compiled DLL into your mods folder, which can be found in your Steam installation:
+
+- **Windows**: `C:\Program Files (x86)\Steam\steamapps\common\Hollow Knight\hollow_knight_Data\Managed\Mods\`.
```
Comment (Extremelyd1) [inline README.md:13]: Perhaps a mention that this is only the case if Hollow Knight is installed to the default steam location. Moreover, the aforementioned doesn't hold when Steam is installed on a differently named volume (e.g. `D:\`).
Another option would be to abstract away from that and call it the Steam install folder, although this is less straightforward for less tech-savvy people.

File: README.md [hunk 3]
```diff
@@ -8,7 +8,15 @@ However, as it seemed to be discontinued, I decided to rework it entirely and ad
...
+
+- **Mac**: `~/Library/Application Support/Steam/steamapps/common/Hollow Knight/hollow_knight.app/`, then click "open package contents" and `content -> resources -> data -> managed -> mods`
+
+- **Linux**: `~/.local/share/Steam/steamapps/common/Hollow Knight/`
```
Comment (Extremelyd1) [inline README.md:17]: I believe this should be suffixed with `hollow_knight_Data/Managed/Mods/`. And the comment below can be removed.

**Review by Extremelyd1 — CHANGES_REQUESTED** [review]
Some general and some nitpicky things that I'd like to see changed.

File: README.md (hunk 2 above)
Comment (Ruttie2006) [reply README.md:13]: hmm afaik the only difference for a D:\ drive would be the letter at the start. not sure tho

**Review by Extremelyd1 — APPROVED** [review]

**Pull request merged**